api/database/seeders/DatabaseSeeder.php

```diff
@@ -270,4 +275,47 @@ private function seedPools()
             }
         }
     }
+
+    private function seedAssessmentResults(Pool $dcmPool)
```

Comment:
Hmm, I see assessment results as being tightly related to the pool candidate.  It makes more sense to me to have the pool candidate factory call the assessment result factory for the rows it needs rather than have it get called from the top level seeder here, I think.

Reply:
I'm not too sure where to best slot it. I added the function in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8302/commits/63eb95c8341d7f37ef9e6b04df29a1071d03347e
But, I feel if you want an assessment step to have multiple associated results, then seeding from pool candidate isn't the best path.

Reply:
I think it makes sense to be attached from a pool candidate way, but seeding like so makes it easier to have a pool with multiple assessments that have multiple associated results. So, the way that would fill in your assessment tracker dashboard with a bunch of data